sdks/java/core/src/main/java/org/apache/beam/sdk/transforms/DoFn.java

```diff
@@ -440,6 +442,76 @@ public BoundedWindow window() {
   }
 
   /**
+   * Annotation for declaring and dereferencing timers.
+   *
+   * <p><i>Not currently supported by any runner.</i>
+   *
+   * <p>To declare a timer, create a field of type {@link TimerSpec} annotated with a {@link
+   * TimerId}. To use the cell during processing, add a parameter of the appropriate {@link Timer}
+   * subclass to your {@link ProcessElement @ProcessElement} method, and annotate it with {@link
```

Comment:
See my prior question to you - I could delete this comment entirely, replacing with "not supported; don't look here" to avoid any confusion or I could leave it as a description of the future API. I don't care that much.